src/test/software/amazon/qldb/integrationtests/StatementExecutionIntegTest.java

```diff
@@ -185,23 +185,23 @@ public void execute_CreateIndex_IndexIsCreated() {
         String searchQuery = String.format(
             "SELECT VALUE indexes[0] FROM information_schema.user_tables WHERE status = 'ACTIVE' AND name ='%s'",
             Constants.TABLE_NAME);
-        String searchValue = driver.execute(
+        Result searchResult = driver.execute(
             txn -> {
-                Result result = txn.execute(searchQuery);
-
                 // Extract the index name by querying the information_schema.
```

Comment:
Could you move this comment to retrieving the value in the buffered result?

Reply:
Done.